source/configuration_manual/mail_location/index.rst

```diff
@@ -1,299 +1,367 @@
+                   * ``Maildir++``: The default used by
+                     :ref:`Maildir <maildir_mbox_format>`.
 
-================= ==============================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
-INDEX               specifies the location of ``Index files``.
+                   * ``fs``: The default used by
+                     :ref:`mbox <mbox_mbox_format>` and
+                     :ref:`dbox <dbox_mbox_format>`.
 
-                    * ITERINDEX: Perform mailbox listing using the INDEX directories instead of the mail root directories. Mainly useful when the INDEX storage is on a faster storage. It takes no value. (v2.2.32+)
+                   * ``index``: Uses mailbox GUIDs as the directory names.
+                     The mapping between mailbox names and GUIDs exists in
+                     ``dovecot.list.index*`` files.
 
-INBOX               specifies the location of the ``INBOX path``.
+``NO-NOSELECT``    Automatically delete any ``\NoSelect`` mailboxes that have
+                   no children. These mailboxes are sometimes confusing to
+                   users. Also if a ``\NoSelect`` mailbox is attempted to be
+                   created with ``CREATE box/``, it's created as selectable
+                   mailbox instead. (``LAYOUT=Maildir++`` always behaves this
+                   same way.) (v2.2.32+)
 
-LAYOUT              specifies the directory layout to use:
+``UTF-8``          Store mailbox names on disk using UTF-8 instead of
+                   modified UTF-7 (mUTF-7).
 
-                    * Maildir++: The default used by Maildir format
+``BROKENCHAR``     Specifies an escape character that is used for broken or
+                   otherwise inaccessible mailbox names. If mailbox name
+                   can't be changed reversibly to UTF-8 and back, encode the
+                   problematic parts using ``<broken_char><hex>`` in the
+                   user-visible UTF-8 name. The broken_char itself also has
+                   to be encoded the same way. This can be useful with
+                   :ref:`imapc_mbox_format` to access mailbox names that
+                   aren't valid mUTF-7 charset from remote servers, or if the
+                   remote server uses a different hierarchy separator and has
+                   folder names containing the local separator. (v2.2.32+)
 
-                    * fs: The default used by mbox and dbox formats
+		   .. versionchanged:: v2.3.14 Conflicting separators are
```

Comment:
looks like too little indentation?

Reply:
It renders fine for me...

Reply:
oh, it's because you're using tabs here but spaces elsewhere. for consistency maybe better to change to spaces.

Reply:
fixed